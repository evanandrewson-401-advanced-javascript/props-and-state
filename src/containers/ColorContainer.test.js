import React from 'react';
import { mount } from 'enzyme';
import ColorContainer from './ColorContainer';

describe('ColorContainer component', () => {
  it('renders ColorContainer', () => {
    const wrapper = mount(<ColorContainer />);
    expect(wrapper).toMatchSnapshot();
  });
});