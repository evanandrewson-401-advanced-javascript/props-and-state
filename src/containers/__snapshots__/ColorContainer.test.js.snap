// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorContainer component renders ColorContainer 1`] = `
<ColorContainer>
  <ColorDisplay
    backgroundColor="black"
  >
    <div
      style={
        Object {
          "backgroundColor": "black",
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </ColorDisplay>
  <ColorPicker
    colors={
      Array [
        "red",
        "green",
        "black",
        "yellow",
      ]
    }
    selectColor={[Function]}
  >
    <button
      key="red"
      onClick={[Function]}
    >
      red
    </button>
    <button
      key="green"
      onClick={[Function]}
    >
      green
    </button>
    <button
      key="black"
      onClick={[Function]}
    >
      black
    </button>
    <button
      key="yellow"
      onClick={[Function]}
    >
      yellow
    </button>
  </ColorPicker>
</ColorContainer>
`;
